.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "semi": false
+}
```

Comment:
I'm not a fan fo prettier code style, so if you could just stick to eslint and limit changes to the places that you really changed, it would be great!

Reply:
No problem